src/components/SearchTextareaField/SearchTextareaField.test.js

```diff
@@ -19,11 +19,15 @@ const searchableIndexes = [{
 }];
 
 const onChange = jest.fn();
+const onSubmitSearch = jest.fn();
+
+const onKeyDown = () => onSubmitSearch();
 
 const renderSearchTextareaField = (props = {}) => render(
   <SearchTextareaField
     id="test-search-textarea-field"
     onChange={onChange}
+    onKeyDown={onKeyDown}
```

Comment:
You already removed this prop from this component, so please remove it from here too.